src/app/containers/ConsentBanner/Banner/cookie.amp.jsx

```diff
@@ -0,0 +1,268 @@
+/* eslint-disable react/no-danger */
```

Comment:
Is this disable needed, I can't see any dangerously set html?

Reply:
Good spot. I have no idea where that came from, but have pruned it.